srcpkgs/base-files/files/vkpurge

```diff
@@ -3,7 +3,7 @@
 # A simple script to remove old kernel files/modules.
 # Brought to you by yours truly Juan RP in the Public Domain.
 #
-: ${progname:=$(basename $0)}
+: "${progname:="$(basename "$0")"}"
```

Comment:
I would just hardcode the program name. If kept, basename can be replaced with `"${0##*/}"`.